internal/client/argument.go

```diff
@@ -0,0 +1,20 @@
+package client
+
+type Arg struct {
+	CommandName string
+	Args        []string
+}
```

Comment:
```suggestion
import "strings"

type Arg struct {
	commandName string
	args        []string
}

func NewArg(message string) *Arg {
	split := strings.Split(message, " ")
	return &Arg {
		commandName: split[0],
		args: split[1:],
	}
}
```

フィールドは非公開にして、文字列から `Arg` を作る手段だけを提供するべき
他のレシーバとかも同様に直してね

Reply:
他のファイルからアクセスできるのもおかしいので、`Arg` 用の `package` を作ってもいいかもしれない